custom/enikshay/tests/test_voucher_counts.py

```diff
@@ -49,7 +59,17 @@ def test_thresholds(self):
         # It should hit 30 now, and should flp the order of the above two
         self.make_voucher(prescription, 7, datetime(2017, 1, 3))
         # This one should trigger both the 60 and 90 thresholds
-        self.make_voucher(prescription, 70, datetime(2017, 1, 4))
+        voucher = self.make_voucher(prescription, 70, datetime(2017, 1, 4))
+        self.assertEqual(
+            EpisodeVoucherUpdate(self.domain, self.cases['episode']).get_prescription_total_days(),
+            {
+                "prescription_total_days": 29 + 1 + 7 + 70,
+                "prescription_total_days_threshold_30": '2017-01-02T00:00:00.000000Z',
+                "prescription_total_days_threshold_60": '2017-01-04T00:00:00.000000Z',
+                "prescription_total_days_threshold_90": '2017-01-04T00:00:00.000000Z',
+            }
+        )
+        self.approve_voucher(voucher)  # approving the voucher shouldn't change anything
```

Comment:
This diff is misleading - the `assertEqual` right below this is the new one.  This test and the other one I added fail without the code change here.